package.json

```diff
@@ -56,7 +56,8 @@
     "react-helmet": "^6.1.0",
     "react-query": "^3.39.1",
     "siwe": "^1.1.6",
-    "ulid": "^2.3.0"
+    "ulid": "^2.3.0",
+    "uuid": "^9.0.0"
```

Comment:
We might consider using the `ulid` library here instead of importing a new `uuid` library.  Not a blocker.